zipkin/src/main/scala/org/http4s/zipkin/interpreters/collector/Http.scala

```diff
@@ -0,0 +1,36 @@
+package org.http4s.zipkin.interpreters.collector
+
+import argonaut.Argonaut._
+import org.http4s.Uri.RegName
+import org.http4s.client.Client
+import org.http4s.zipkin.algebras.{CollectorInterpreter, CollectorOp}
+import org.http4s.zipkin.interpreters._
+import org.http4s.{MediaType, Method, Request, Uri}
+
+import scalaz.concurrent.Task
+
+final case class Http(client: Client) extends CollectorInterpreter {
+  override def apply[A](fa: CollectorOp[A]): Task[A] = fa match {
+    case CollectorOp.Send(zipkinInfo) =>
+      val uri = Uri(
+        authority = Option(Uri.Authority(
+          host = RegName("localhost"),
+          port = Option(9411)
+        )),
+        path = "/api/v1/spans"
+      )
+      println(uri)
+      val payload = List(zipkinInfo).asJson.spaces2
+      println(payload)
+      val request = Request(Method.POST, uri)
+        .withContentType(Option(MediaType.`application/json`))
+        .withBody(payload)
+
+      client
+        .fetch(request)(response => Task.now(()))
+        .runAsync(_ => ())
```

Comment:
This is the equivalent of an empty catch block.  I don't understand why we're not just returning this task?